### PR TITLE
New package: cfm-0.6.3

### PR DESCRIPTION
**Pull request by willeccles**
Added package cfm at version 0.6.3. cfm (cactus file manager) is a simple TUI file manager with no dependencies. I will be maintaining the package.

### CODE REVIEW

File: srcpkgs/cfm/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'cfm'
+pkgname=cfm
+version=0.6.3
+revision=1
+build_style=gnu-makefile
+make_use_env=compliant
```
Comment (ahesford) [inline srcpkgs/cfm/template:6]: Just set to "yes" here, "compliant" is just different for the sake of differences.

**Review by ahesford — CHANGES_REQUESTED** [review]
Squash commits into one

File: srcpkgs/cfm/template (hunk 1 above)
Comment (willeccles) [reply srcpkgs/cfm/template:6]: I was following what st does. Will update.

**Comment by willeccles** [comment]
Commits have been squashed; made requested change.

**Pull request merged**